Source/utils/surface_to_pcx.cpp

```diff
@@ -15,13 +15,15 @@
 namespace devilution {
 namespace {
 
-tl::expected<void, std::string> CheckedFWrite(const void *ptr, size_t size, FILE *out)
+tl::expected<void, std::string> CheckedFWrite(const void *ptr, size_t size, SDL_RWops *out)
 {
-	if (std::fwrite(ptr, size, 1, out) != 1) {
-		const char *errorMessage = std::strerror(errno);
+	if (out->write(out, ptr, size, 1) != 1) {

```

Comment:
This doesn't really matter, but.. it's more consistent with `SDL_RWclose()`, and the first parameter feels like it makes more sense. :shrug:
```suggestion
	if (SDL_RWwrite(out, ptr, size, 1) != 1) {
```

Reply:
Done